src/screens/AppletsScreen/index.tsx

```diff
@@ -1,24 +1,32 @@
 import { FC } from 'react';
+import { ImageBackground, StyleSheet } from 'react-native';
 
-import { colors } from '@app/shared/lib';
-import { Text, YStack } from '@shared/ui';
+import { AppletList } from '@app/entities/applet';
+import { Box, ScrollView } from '@shared/ui';
 
 const AppletsScreen: FC = () => {
   return (
-    <YStack
-      bg="$secondary"
-      flex={1}
-      alignItems="center"
-      justifyContent="center">
-      <Text color={colors.tertiary} fontSize={18}>
-        User is logged in
-      </Text>
-
-      <Text color={colors.tertiary} fontSize={18}>
-        This is mock for applet list
-      </Text>
-    </YStack>
+    <Box bg="$secondary" flex={1}>
+      <ImageBackground
```

Comment:
I assume this background image will be used on a lot of screens. Perhaps, that the default image for all screens, except the authentication ones.
I'd suggest using this approach: https://stackoverflow.com/a/70117826

Let's create a BackgroundImage component that wraps the children in the `ImageBackground`. Then we'll wrap the NavigationContainer into BackgroundImage.